study.md

```diff
@@ -134,15 +134,15 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP is a system of rules governing data transfer between computers across the web
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
+In the context of our lesson, both are computers, but the client is used by and end-user to access remote resources and applications, while a server is responsible for serving centralized resources and applications to client computers (or other servers, I guess, which eventually pass the resources to a client computer).  The client sends requests and receives responses, while the server receives requests and sends responses.
```

Comment:
Good, the client can refer to the program or browser you are using as well (Chrome, Safari, etc.)